packages/rich-text/README.md

```diff
@@ -220,6 +220,8 @@ _Returns_
 
 ### getTextContent
 
+> **Deprecated** Use the text property instead.
```

Comment:
noted in Slack, but I wonder if this could still be a useful method to have, since it gives us a little more freedom to continue to mess around with the internal state representation without accidentally exposing something to the outside that constrains us.

turning `.text` into a getter also works, but having the method is perhaps a bit simpler when thinking through how to make those changes.

Reply:
Sure, we can leave it here I guess